tests/e2e/test_app.py

```diff
@@ -28,25 +29,37 @@
 )
 
 
+@pytest.fixture(scope="module", params=[None, TEST_NAMESPACE])
+def namespace(request):
+    return request.param
+
+
+@pytest.fixture(scope="module")
+def namespace_prefix(namespace):
+    return f"{namespace}/" if namespace is not None else ""
+
+
+@pytest.fixture(scope="module")
+def index_name(testrun_uid: str, namespace):
```

Comment:
Actually - creating the index _shouldn't_ depend on the `namespace` fixture.  
We should create a single index, then repeat all tests against the same index - once with namespace and once without namespace.

Reply:
Right fixed that!